user-service/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=uk.gov.gchq.palisade.example.config.ApplicationConfiguration
```

Comment:
This should be the application configuration in services rather than the example one

Reply:
The services config already has the @EnableAutoConfiguration annotation applied to it so changing this to the service application config creates a circular import issue when starting up the service.
This is how the example config beans are loaded into the service.

Reply:
Moved spring.factories to example-library.